src/fides/cli/commands/export.py

```diff
@@ -10,14 +10,18 @@
 from fides.cli.utils import with_analytics
 from fides.core import export as _export
 from fides.core import parse as _parse
+from fides.core.utils import echo_red
 
 
-@click.group(name="export")
+@click.group(name="export", deprecated=True)
 @click.pass_context
 def export(ctx: click.Context) -> None:
     """
     Export Fides data maps.
     """
+    echo_red(
+        "If you wish to use the 'export' command in future releases, you must use 'fidesplus'"
```

Comment:
If possible it might be nice to add something here about how this can still work without `fidesplus` or even just why. Maybe something like: 
```
Improved `export` functionality will exist in `fidesplus` in future releases. Extracting resource data is still possible via the API.
```

Open to ideas or leaving it 🤷🏽

Reply:
oh i like that! i didn't feel too good about my text but left it there with the implicit assumption that someone more thoughtful (i.e., you) would have better wording 😄 

i'll adjust and if others want to tweak, please let us know in this thread!

Reply:
updated, here's output with the update:

<img width="962" alt="image" src="https://github.com/ethyca/fides/assets/9750285/3ed36bc8-ea17-4c91-a217-6c0765bef52c">